cpp/src/io/csv/csv_reader.cu

```diff
@@ -888,20 +892,21 @@ __global__ void countRecords(char *data, const char terminator, const char quote
 gdf_error launch_storeRecordStart(raw_csv_t * csvData) {
 
 	int blockSize;		// suggested thread count to use
-	int minGridSize;	// minimum block count required
-	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, storeRecordStart) );
+ 	int minGridSize;	// minimum block count required
+ 	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, storeRecordStart) );
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.